src/lightning/pytorch/CHANGELOG.md

```diff
@@ -284,6 +284,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Removed support for passing a scheduling dictionary to `Trainer(accumulate_grad_batches=...)` ([#16729](https://github.com/Lightning-AI/lightning/pull/16729))
 
+
+- Removed support for `DataParallel` (`strategy='dp'`) ([#16748](https://github.com/Lightning-AI/lightning/pull/16748))
```

Comment:
Can you mention the removed classes in `src/lightning/pytorch/overrides/data_parallel`